src/biokbase/narrative/tests/test_appmanager.py

```diff
@@ -600,7 +542,46 @@ def run_func():
         "biokbase.narrative.jobs.appmanager.auth.get_agent_token",
         side_effect=mock_agent_token,
     )
-    def test_run_app_bulk_good_inputs(self, auth, c):
+    def test_run_app_bulk__dry_run(self, auth, c):
```

Comment:
Moved function up and transplanted some of the logic from "wet run" test

Reply:
Comprehensively tested the results of the "dry run" method (so it's safe to remove the other attribute testing in the wet run version)